src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -240,21 +250,13 @@ void KviMenuBar::setupMainPopup(QMenu *pop)
 	m_pDisconnectAction = main->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Quit)),__tr2qs("Disconnect"),this,SLOT(actionTriggered(bool)));
 	m_pDisconnectAction->setData(KVI_INTERNALCOMMAND_QUIT);
 
-	main->addSeparator();
-
 	// FIXME: Add a "Dock to tray" icon if the tray is not visible (or show tray icon or whatever)
 
 // Qt/Mac creates a Quit item on its own <= this is bad
 #ifndef COMPILE_ON_MAC
 	main->addSeparator();
 
-	main->addAction(
```

Comment:
I actually think this bit is only cleanup, cant see any difference at first glace